src/lib/plot/hint.js

```diff
@@ -215,15 +269,18 @@ class Hint extends PureRenderComponent {
 
   render() {
     const {
+      className,
+      style,
```

Comment:
Will drop this addition since this requires more broader discussion for the package.